base/src/main/java/org/mozilla/jss/netscape/security/x509/OIDMap.java

```diff
@@ -166,6 +173,10 @@ private static void loadClassDefault(Properties props) {
                   "org.mozilla.jss.netscape.security.x509.SubjectDirAttributesExtension");
         props.put(EXT_KEY_USAGE,
                   "org.mozilla.jss.netscape.security.extensions.ExtendedKeyUsageExtension");
+        props.put(EXT_INHIBIT_ANY_POLICY,
+                "org.mozilla.jss.netscape.security.extensions.InhibitAnyPolicyExtension");
+        props.put(OCSP_NO_CHECK,
+                "org.mozilla.jss.netscape.security.extensions.OCSPNoCheckExtension");
```

Comment:
To ensure that the class names and packages are correct we probably should use `InhibitAnyPolicyExtension.class.getName()` and `OCSPNoCheckExtension.class.getName()`.